Kwf/User/Auth/ActivationFields.php

```diff
@@ -0,0 +1,58 @@
+<?php
+class Kwf_User_Auth_ActivationFields extends Kwf_User_Auth_Abstract implements Kwf_User_Auth_Interface_Activation
```

Comment:
Please comment what this class does in terms of "token times out", "token is only valid one time or more often"..